site/_posts/2017-10-30-crossword-lwimiq.html.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Crossword LWiMIQ"
+author: d-m-u
+date: 2017-10-30
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+## A Puzzle for your Perusal
+
+Any puzzles, correctly completed and returned to me before next week's blog, will enter you into a drawing for a prize!
+
+![crossword](/site/assets/images/blog/crossword.jpg)
```

Comment:
Due to the way relative paths are handled, "site" shouldn't be in the path (I know it'll cause the image not to show up in the preview but instead it will show up on the published post).

i.e. `![crossword](/assets/images/blog/crossword.jpg)`